### ISSUE TITLE
quarto <subcommand> --help does not work for 'preview' and 'run'

### ISSUE THREAD

**Issue by HenrikBengtsson** [quarto-dev/quarto-cli#1333]
### Bug description

Despite `quarto render --help`, `quarto serve --help`, etc. work for most sub-commands, it does not work for:

* [ ] `quarto preview --help`
* [ ] `quarto run --help`

Specifically,

```sh
$ quarto preview --help
ERROR: /home/hb is not a website or book project

$ quarto run --help
ERROR: quarto run: script '--help' not found
```

This is on an up-to-date Ubuntu 20.04 machine.

Possibly related to https://github.com/quarto-dev/quarto-cli/issues/598

### `quarto check` Output

```
$ quarto check

[✓] Checking Quarto installation......OK
      Version: 0.9.644
      Path: /home/hb/shared/software/CBI/quarto-0.9.644/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.8.10
      Path: /usr/bin/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

[✓] Checking R installation...........OK
      Version: 4.2.1
      Path: /home/hb/shared/software/CBI/R-4.2.1-gcc9/lib/R
      LibPaths:
        - /home/hb/R/x86_64-pc-linux-gnu-library/4.2-CBI-gcc9
        - /home/hb/R/x86_64-pc-linux-gnu-library/4.2-CBI-gcc9/_bioc-3.15
        - /home/hb/shared/software/CBI/R-4.2.1-gcc9/lib/R/library
      rmarkdown: 2.14

[✓] Checking Knitr engine render......OK
```

### `quarto tools check` Output

```
$ quarto tools check
(-) Inspecting toolsDownload https://deno.land/x/cache@0.2.12/directories.ts
(\) Inspecting toolsDownload https://deno.land/x/cache@0.2.12/deps.ts
[✓] Inspecting tools

Tool         Status            Installed     Latest  
chromium     Not installed     ---           869685  
tinytex      Not installed     ---           v2022.07
```

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] documented the quarto version you're running, by pasting the output from running `quarto check` in the "Quarto Check Output" text area?
- [X] documented the version of the quarto tools you're running, by providing the output from running `quarto tools check` in the "Quarto Tools Check Output" text area?
- [ ] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [X] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by jjallaire** [comment]
Yes this is because these commands are processed specially (w/r/t argument forwarding). We'll resolve this post v1.0

**Comment by SamEdwardes** [comment]
In case this helps anyone in the future you can find the options here for `quarto preview`:

https://github.com/quarto-dev/quarto-cli/blob/main/src/command/preview/cmd.ts

**Comment by SamEdwardes** [comment]
> You can also just type `quarto preview help`

Thank you, JJ! I did not know that was an option. I thought `quarto preview --help` would be the only way and that is not going to be fixed until post v1.0

**Comment by jjallaire** [comment]
Thanks again for reporting this, fixed here: https://github.com/quarto-dev/quarto-cli/commit/1908f728ea17c0fe1f838016beb37eb21d4b86cd
